examples/install_with_clone.sh

```diff
@@ -71,34 +90,26 @@ mysql < sql/0.5.0.sql
 mysql < sql/0.6.0.sql
 mysql < sql/0.7.0.sql
 
+PASSWD=""
 
 cp examples/lxd_manager.conf /etc/apache2/sites-available/
 
-pm2 start node/events.js
+pm2 start node/events.js || exit $?
 
-pm2 startup
+pm2 startup || exit $?
 
-pm2 save
+pm2 save || exit $?
 
 # Add cron job for gathering data
-crontab -l | { cat; echo "*/5 * * * * php /var/www/LxdMosaic/src/cronJobs/fleetAnalytics.php"; } | crontab -
+crontab -l 2>/dev/null | { cat; echo "*/5 * * * * php /var/www/LxdMosaic/src/cronJobs/fleetAnalytics.php"; } | crontab -
 crontab -l | { cat; echo "*/1 * * * * php /var/www/LxdMosaic/src/cronJobs/hostsOnline.php"; } | crontab -
 
-# Enable required apache mods
-a2enmod ssl
-a2enmod headers
-a2enmod rewrite
-a2enmod proxy
-a2enmod proxy_wstunnel
-a2enmod proxy_wstunnel
-a2enmod proxy_http
-a2enmod proxy_balancer
-
 # Enable site
 a2ensite lxd_manager
 
-systemctl restart apache2
+systemctl restart apache2 || exit $?
 
+IP=`ip route get 1 | sed -n 's/^.*src \([0-9.]*\) .*$/\1/p'`
```

Comment:
That's naughty - i like it

Reply:
Well if you want to bind apache to an ip/interface that is not used for your default gateway then you need to edit the apache site config file anyway so...